src/test/java/e2e/pages/ProfilePage.java

```diff
@@ -0,0 +1,71 @@
+package e2e.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+import java.util.NoSuchElementException;
+
+public class ProfilePage extends PageBase {
+    public ProfilePage(WebDriver driver) {
+        super(driver);
+    }
+
+    @FindBy(xpath = "//*[@title='Delete']")
+    WebElement deleteButton;
+    @FindBy(xpath = "//*[@id='closeSmallModal-ok']")
+    WebElement deleteOkButton;
+
+    public void waitForLoading() {
+        getWait().forVisibility(deleteButton);
+    }
+
+//    @FindBy(xpath = "//*[@id='gotoStore']")
+//    WebElement goToStoreButton;
+
+    //    public void clickGoToBookStoreButton() {
+//        try {
+//            getWait().forVisibility(goToStoreButton);
+//            goToStoreButton.click();
+//        } catch (NoSuchElementException e) {
+//            System.out.println("Element not found : " + e.getMessage());
+//        }
+//    }
+//}
+    public void clickGoToBookStoreButton() {
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));
+        try {
+            WebElement goToStoreButton = wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//*[@id='gotoStore']")));
```

Comment:
need to describe waitForLoading and move to waitForLoading method all waiting options (forClickble, forVisible)

Reply:
made